lib/pbio/sys/program_load.h

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2022 The Pybricks Authors
+
+#ifndef _PBSYS_SYS_PROGRAM_LOAD_H_
+#define _PBSYS_SYS_PROGRAM_LOAD_H_
+
+#include <pbio/error.h>
+
+#include <pbsys/config.h>
+#include <pbsys/main.h>
+
+#include <stdint.h>
+
+#if PBSYS_CONFIG_PROGRAM_LOAD
```

Comment:
In what case would this option be disabled?

Reply:
Any build that has `sys` enabled but does not receive/load programs, like the virtual hub.

It could make sense to enable sys in full on the virtual hub so it works more like the other embedded hubs, but that's something for another day.